rm-main/src/ui/tabs/torrents/mod.rs

```diff
@@ -41,7 +40,7 @@ impl TorrentsTab {
         let table_manager = Arc::new(Mutex::new(TableManager::new(ctx.clone(), table)));
         let stats = Arc::new(Mutex::new(None));
         let free_space = Arc::new(Mutex::new(None));
-        let bottom_stats = BottomStats::new(stats, free_space, table_manager.clone());
+        let bottom_stats = BottomStats::new(stats, free_space, Arc::clone(&table_manager));
```

Comment:
does calling `.clone` on an `Arc` not do the same as `Arc::clone` ?

Reply:
yes, but it's more explicit that you don't actually make an operation that is costly, you are just incrementing the reference count